### ISSUE TITLE
Adding more args to json files of pretrained models 

### ISSUE THREAD

**Issue by JanisHe** [seisbench/seisbench#307]
Hi Jannes,

I noted, that it might be hard to fully understand how some models have been trained. E.g. for the STEAD PhaseNet one knows that the model was trained for 100 epochs with a learning rate of 0.01. However, I would also mention the batch size, sigma and the sampling rate of the data used, since for application and/or fine tuning of these models it might be a relevant information for the users. E.g. diting was trained on 50 Hz data set but this relevant information is not included (or I did not see it) somewhere. I think for a correct application these arguments could be important.

What do you think about it?

Regards,
Janis

**Comment by yetinam** [comment]
Hi @JanisHe ,

thanks for the suggestion. I general, the weights metadata has two different kinds of information. The docstring, which is purely for documentation and not automatically parsed and the parameters that are used by seisbench (component order, sampling rate, normalization, ...). For the parameters, they are often only specified if they don't match the default values (which in hindsight probably wasn't the best choice). In general, I agree that a more detailed documentation might be useful. The details you mention are usually anyhow provided in the papers referenced but it might be good to have them in one place. I'm not sure it makes sense to do this retroactively for the existing weights but it should be considered when adding a new set of weights.

Regarding DiTing, where did you find it's trained on 50 Hz? That would indeed be an issue, because if no sampling rate is provided, SeisBench assumes the default of 100 Hz.

**Comment by JanisHe** [comment]
Hi Jannes,
 
the 50 Hz are written in the Paper (https://www.equsci.org.cn/en/article/doi/10.1016/j.eqs.2023.03.001). 
![diting](https://github.com/user-attachments/assets/3f062f8a-d7af-483e-8d2c-8eaf856740e7)

I totally agree that it might be hard to add these metadata to existing models, but taking this into account for future models is good way.

**Comment by yetinam** [comment]
That sounds convincing... Thanks for pointing that out! I've asked the original authors to verify and will fix the metadata once this has been confirmed.

**Comment by yetinam** [comment]
I've updated the files for DiTing to use 50 Hz sampling rate and added a warning in 05cb1ce5f2022df17a897c12b8f8b502cb0ed304 .
